tasks/main.yml

```diff
@@ -20,7 +20,7 @@
   tags:
     - haproxy
 
-- name: Start Marathon service
-  service: name=marathon state=restarted enabled=yes
+- name: Enable Marathon service
+  service: name=marathon enabled=yes
```

Comment:
yeah was thinking about setting state=started (and ok for me) 
finally I opted to leave it out, as the handler will ensure the service is started in a ll cases except for the case that marathon was (somehow) not running and no config was changed  (in that case you could prefer the service state is not changed, on the other side, in case the service ended accidentally you could prefer that a (regular) ansible deployment run  ensures the service is running on the defined inventory hosts)